From: Egbert Eich <eich@suse.com>
Date: Tue Jun 27 20:18:53 2017 +0200
Subject: Include grp.h
Patch-mainline: Not yet
Git-repo: https://github.com/chaos/mrsh
Git-commit: db3761b4977a2d9cfb43de1b5868424d8e8101ee
References: 

This gets rid of an implicit declaration warning in mrlogind/auth.c.

Signed-off-by: Egbert Eich <eich@suse.com>
Signed-off-by: Egbert Eich <eich@suse.de>
---
 mrlogind/auth.c | 1 +
 1 file changed, 1 insertion(+)
diff --git a/mrlogind/auth.c b/mrlogind/auth.c
index 30f2dd3..827968e 100644
--- a/mrlogind/auth.c
+++ b/mrlogind/auth.c
@@ -66,6 +66,7 @@
 
 #include <sys/types.h>
 #include <pwd.h>
+#include <grp.h>
 
 #include "mrlogind.h"
 
